ynput/OpenPype
# Nuke: publisher is offering review on write families on demand

## Changelog Description
Original idea where reviewable toggle will be offered in publisher on demand is fixed and now `review` attribute can be disabled in settings.

## Additional info
Removed duplicity in class functions. `get_instance_attr_defs` didn't need to be overridden with child instance. 

## Testing notes:
1. go to your testing project studio settings `project_settings/nuke/create/CreateWritePrerender/instance_attributes` and remove `reviwable`. Save settings 
2. Open nuke at testing project workfile
3. open Publisher and create Prerender node
4. Notice that `Review` instance attribute is not shown


# Dependency
Will work only if https://github.com/ynput/OpenPype/pull/4776 is merged before.
